cms/static/js/features_jsx/studio/index.js

```diff
@@ -65,18 +65,22 @@ class CourseOrLibraryListing extends React.Component {
 
 export class StudioCourseIndex {
     constructor(selector, context, allowReruns) {
+      if (document.querySelector(selector) !== null) {
```

Comment:
@sarahkf in both `StudioCourseIndex` and `StudioLibraryIndex`, you can simplify by storing the output of `document.querySelector(selector)` in a variable so you don't have to call it twice.